test/cucumber/ParkingSpotsReportedByUsers.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Parking spots reported by users
+ As a user of the system
+ I want to inform the system which parking spot my car is parked
+ So that the system remain up to date
+
+Scenario: Report that parked in the spot indicated by the system web
+ Given that the system indicated to the user "x" the parking spot "20"
+ When the user "x" is parking in the parking spot "20"
+ And fills in the system that parked in the parking spot �20�
+ Then the system informs that the state of the parking spot "20" was changed with sucess
```

Comment:
Enquanto isso é um problema bem menor, o texto poderia estar escrito no futuro algo como

    Then the system will inform that the state of the parking spot "20" was changed with sucess

Mas ainda sim existe um equívoco de escrita de inglês, melhoria seria

    Then the system will inform that the state of the parking spot "20" was changed with success

Acredito que não ficou claro qual é a mudança de estado que será feita na vaga de estacionamento para ser testado se ocorreu com sucesso.